framework/generated/generated_vulkan_replay_consumer.cpp

```diff
@@ -49,25 +49,25 @@ void VulkanReplayConsumer::Process_vkCreateInstance(
     HandlePointerDecoder<VkInstance>*           pInstance)
 {
     if (!pInstance->IsNull()) { pInstance->SetHandleLength(1); }
-    VulkanInstanceInfo handle_info;
+    InstanceInfo handle_info;
```

Comment:
I assume you fixed this later since it shouldn't compile until you do.  You should likely squash the fix for this with this change or else the tree won't be bisect-able.

Reply:
yes, this was fixed by a rebase in latest commit. we normally stash our PR-commits when merging, so that should remove this intermediate state.